feature-libs/cart/import-export/components/export-entries/export-entries.component.spec.ts

```diff
@@ -1 +1,154 @@
+    value: 23.5,
+  },
+  cancellableQuantity: 0,
+  entryNumber: 1,
+  product: {
+    availableForPickup: false,
+    baseOptions: [],
+    categories: [
+      {
+        code: '1358',
+        name: 'Measuring & Layout Tools',
+        url: '/Open-Catalogue/Tools/Measuring-%26-Layout-Tools/c/1358',
+      },
+      {
+        code: 'brand_4434',
+        name: 'Ednet',
+        url: '/Brands/Ednet/c/brand_4434',
+      },
+    ],
+    code: '3803058',
+    images: {
+      PRIMARY: {
+        zoom: {
+          altText: 'PC Service Set Professional',
+          format: 'zoom',
+          imageType: ImageType.PRIMARY,
+          url: 'imageUrl',
+        },
+        product: {
+          altText: 'PC Service Set Professional',
+          format: 'product',
+          imageType: ImageType.PRIMARY,
+          url: 'imageUrl',
+        },
+        thumbnail: {
+          altText: 'PC Service Set Professional',
+          format: 'thumbnail',
+          imageType: ImageType.PRIMARY,
+          url: 'imageUrl',
+        },
+        cartIcon: {
+          altText: 'PC Service Set Professional',
+          format: 'cartIcon',
+          imageType: ImageType.PRIMARY,
+          url: 'imageUrl',
+        },
+      },
+    },
+    manufacturer: 'Ednet',
+    name: 'PC Service Set Professional',
+    purchasable: true,
+    stock: {
+      stockLevel: 365,
+      stockLevelStatus: 'inStock',
+    },
+    url:
+      '/Open-Catalogue/Tools/Measuring-%26-Layout-Tools/PC-Service-Set-Professional/p/3803058',
+    slug: 'pc-service-set-professional',
+    nameHtml: 'PC Service Set Professional',
+  },
+  quantity: 2,
+  returnableQuantity: 0,
+  totalPrice: {
+    currencyIso: 'USD',
+    formattedValue: '$47.00',
+    priceType: PriceType.BUY,
+    value: 47,
+  },
+  updateable: true,
+};
+
+class MockExportEntriesService {
+  getEntries() {
+    return of([entry]);
+  }
+}
+
+describe('ExportEntriesComponent', () => {
+  let component: ExportEntriesComponent;
+  let fixture: ComponentFixture<ExportEntriesComponent>;
+
+  beforeEach(
+    waitForAsync(() => {
+      TestBed.configureTestingModule({
+        imports: [
+          StoreModule.forRoot({}),
+          I18nTestingModule,
+          RouterTestingModule,
+        ],
+        providers: [
+          {
+            provide: RoutingService,
+            useClass: MockRoutingService,
+          },
+          {
+            provide: ExportEntriesService,
+            useClass: MockExportEntriesService,
+          },
+        ],
+        declarations: [ExportEntriesComponent],
+      }).compileComponents();
+    })
+  );
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(ExportEntriesComponent);
+    component = fixture.componentInstance;
+
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('should display export to csv link and run downloadCsv on click', () => {
+    spyOn(component, 'downloadCsv');
```

Comment:
spying component methods dont have too much sense (especially in this case). We should do it on external stuff and mock responses possibly
```suggestion
    spyOn(exportService, 'dataToCsv');
```

Reply:
We don't need even this spy here for test such scenario.